demos/tutorial/indexed_fasta_io/solution1.cpp

```diff
@@ -25,7 +25,7 @@ int main(int argc, char const ** argv)
     if (!save(faiIndex, toCString(faiFilename)))
     {
         std::cerr << "ERROR: Could not write the index to file!\n";
-        return 1;
+        return 0;
     }
 
     std::cout << "Index file " << faiFilename << " was successfully created.\n";
```

Comment:
OK. It returns the control before the execution of this line.
So the stdout should be empty.

>   if (argc != 2)
>     {
>         std::cerr << "USAGE: build_fai FILE.fa\n";
>         return 0;
>     }